include/deal.II/lac/sparse_matrix_ez.h

```diff
@@ -429,12 +429,23 @@ class SparseMatrixEZ : public Subscriptor
    */
 //@{
   /**
-   * Set the element <tt>(i,j)</tt> to @p value. Allocates the entry, if it
-   * does not exist and @p value is non-zero. If <tt>value</tt> is not a
-   * finite number an exception is thrown.
+   * Set the element <tt>(i,j)</tt> to @p value.
+   *
+   * If <tt>value</tt> is not a finite number an exception is thrown.
+   *
+   * The optional parameter <tt>elide_zero_values</tt> can be used to specify
+   * whether zero values should be added anyway or these should be filtered
+   * away and only non-zero data is added. The default value is <tt>true</tt>,
+   * i.e., zero values won't be added into the matrix.
+   *
+   * If anyway a new element will be inserted and it does not exist,
+   * allocates the entry.
+   *
+   * @note: You may need to insert some zero elements to keep a
```

Comment:
I'll let others do the merge if they think it is right (a testcase might help?) but just wanted to note that you don't need the ':' after `@note`. doxygen marks everything up correctly already.